internal/connector/internal/services/connector_types.go

```diff
@@ -189,6 +189,10 @@ func (cts *connectorTypesService) List(listArgs *services.ListArguments) (dbapi.
 		Size: listArgs.Size,
 	}
 
+	dbConn = dbConn.Joins("LEFT JOIN connector_type_channels channels on channels.connector_type_id = connector_types.id")
```

Comment:
This is a good solution to ensuring only supported channels are visible. But it should be ignored if the new variable is not set.